scalameta/scalameta/shared/src/test/scala/scala/meta/tests/prettyprinters/PublicSuite.scala

```diff
@@ -139,7 +140,7 @@ class PublicSuite extends FunSuite {
   test("scala.meta.inputs.Input.File.toString") {
     import java.io._
     import java.nio.charset.Charset
-    val path = AbsolutePath.fromRelative("hello.scala")
+    val path = RelativePath("hello.scala").absolutize(PlatformIO.workingDirectory)
```

Comment:
What do you dislike in the current scheme of things? Would renaming the methods to `toAbsolute` and `toRelative` work for you? Or is it something else?